bakerydemo/base/fixtures/bakerydemo.json

```diff
@@ -12085,6 +12050,28 @@
       "created_at": "2019-06-23T09:08:36.064Z"
     }
   },
+  {
+    "model": "wagtailadmin.editingsession",
+    "pk": 1,
+    "fields": {
+      "user": ["admin"],
+      "content_type": ["wagtailcore", "page"],
+      "object_id": "82",
+      "last_seen_at": "2024-08-06T01:35:42.544Z",
+      "is_editing": false
+    }
+  },
+  {
+    "model": "wagtailadmin.editingsession",
+    "pk": 2,
+    "fields": {
+      "user": ["admin"],
+      "content_type": ["wagtailcore", "page"],
+      "object_id": "82",
+      "last_seen_at": "2024-08-06T01:38:47.215Z",
+      "is_editing": false
+    }
+  },
```

Comment:
These models are new, should these be part of the fixtures? Asking @laymonage or @gasman for input.

Reply:
If we want to add new models, that should be done in a separate PR. For this PR, we should filter the fixture changes to only the ones relevant for the PR.